src/pages/GenerateKeys/Option1.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Heading } from '../../components/Heading';
+import { Text } from '../../components/Text';
+import { Link } from '../../components/Link';
+import { Alert } from '../../components/Alert';
+import { CHAIN_NAME, IS_MAINNET } from '../../utils/envVars';
+import { Paper } from '../../components/Paper';
+import { colors } from '../../styles/styledComponentsTheme';
+
+export const Option1 = ({
+  validatorCount,
+  os,
+}: {
+  validatorCount: number | string;
+  os: string;
+}) => {
+  return (
+    <Paper className="mt20">
+      <Heading level={2} size="small" color="blueMedium" className="mb20">
+        Option 1: Use binary executable file
+      </Heading>
+      <Text weight={500}>Download the binary executable file</Text>
+      <Text>
+        Please find the binary execution file of the latest deposit-cli release
+        on{' '}
+        <Link
+          primary
+          external
+          to="https://github.com/ethereum/eth2.0-deposit-cli/releases/"
+          inline
+        >
+          ethereum/eth2.0-deposit-cli
+        </Link>{' '}
+        GitHub page for your platform.{' '}
+      </Text>
+      <Alert variant="warning" className="my20">
+        <Text weight={500} color="yellowDarkest" className="my10">
+          Please make sure that you are downloading from Ethereum Foundation
+          official GitHub account.
+        </Text>
+      </Alert>
+      <Text weight={500}>
+        Generate deposit keys using the Ethereum Foundation deposit tool
+      </Text>
+      <Text className="mt5">
+        Type the following lines into the terminal window:
+      </Text>
+      <Alert variant="secondary" className="my10">
```

Comment:
I'm unsure if we need this section. When I run the executable deposit from the download on OSX it automatically runs the script. @hwwhww @CarlBeek  should we remove this section? If so for which platforms?
<img width="1084" alt="Screen Shot 2020-07-31 at 5 10 06 PM" src="https://user-images.githubusercontent.com/7051522/89089250-41532080-d351-11ea-8c28-d3503eace3b3.png">

Reply:
Oh interesting. Did you see the "Please choose how many validators you wish to run: Please choose how many validators you wish to run:" automatically?

I'm on macOS, when I clicked the file it, it automatically runs with "deposit; exit" command. But I need to input a new line to get enter the prompt and see "Please choose how many validators you wish to run: Please choose how many validators you wish to run:". (on both bash or zsh)

![image](https://user-images.githubusercontent.com/9263930/89120150-83619b00-d4e6-11ea-8f3e-fcafba78f698.png)